### PR TITLE
Importance of artifacts scrum 41

### PR DESCRIPTION
**Pull request by Memai2023**
Used one branch to style the whole artifact page including checking the buttons to remove the styles done in artifact and only use the global style.

### CODE REVIEW

File: src/Components/Artifacts/Importance/Importance.module.css [hunk 1]
```diff
@@ -14,23 +14,16 @@
...
+    box-shadow: rgba(0, 0, 0, 0.3) 0px 19px 38px, rgba(0, 0, 0, 0.22) 0px 15px 12px;
+    width: 100%;
 }
 
 .navigationButton {
     justify-content: center;
     display: flex;
 }
 
-.navigationButton a {
```
Comment (blombergalex) [inline src/Components/Artifacts/Importance/Importance.module.css:26]: snyggt att du tar bort detta

File: src/Components/Artifacts/SprintReview/SprintReview.module.css [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 .twoColumnTextContainer {
     background-color: var(--artifact-sprint-review-bg-color);
+    box-shadow: rgba(0, 0, 0, 0.56) 0px 22px 70px 4px;
```
Comment (blombergalex) [inline src/Components/Artifacts/SprintReview/SprintReview.module.css]: nice effect:)

**Review by blombergalex — APPROVED** [review]
Det ser riktigt bra ut, snyggt jobbat!

**Comment by Memai2023** [comment]
Thank you for feedback, now made new changes please internal test

**Pull request merged**